docs/source/usage/inventory.rst

```diff
@@ -64,8 +97,7 @@ an API access token in every request.
 
   Check hosts SSL certificate (true) or not (false). Please be aware that disabling SSL certificate
   verification is a security risk and should not be used in a production environment.
-  `Don't say I didn't tell you <https://www.youtube.com/watch?v=1bVy1sLVasY>`_.
-
+  
```

Comment:
perhaps leave this "Don't say I didn't tell you" in two times, for those navigating just to one of the sections...